model_cards/zanelim/singbert/README.md

```diff
@@ -0,0 +1,166 @@
+---
+language: english, chinese, malay
```

Comment:
```suggestion
language: en
```
We now use ISO codes. Also, I don't personally think Chinese/Malay should be added here (cuz Singlish is not often considered a Chinese/Malay dialect)

Reply:
Thanks for your suggestion! Yes i think it makes sense :))